addons/crm/report/crm_opportunity_report_views.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+        <record id="crm_opportunity_report_view_tree" model="ir.ui.view">
+            <field name="name">crm.opportunity.report.tree</field>
+            <field name="model">crm.lead</field>
+            <field name="inherit_id" ref="crm.crm_case_tree_view_oppor"/>
+            <field name="mode">primary</field>
+            <field name="priority">32</field>
```

Comment:
Can you tell me why 32? :p

Reply:
Indeed not really necessary here :sweat_smile:
Removing it :slightly_smiling_face: